src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -245,4 +246,18 @@ namespace AppInstaller::CLI::Workflow
         }
         table.Complete();
     }
+
+    void ShowMultiLineField(Execution::OutputStream& outputStream, StringResource::StringId label, std::string& value)

```

Comment:
I'd put this function in an anonymous namespace at the top of this file, instead of declaring it in the header. I believe the *Flow.h headers contain just WorkflowTasks that can be called with `context << task`.

Reply:
Ohhhhh, that makes sense. I think I see now how they did it